packages/pillarbox-debug-panel/scss/pillarbox-debug-panel.scss

```diff
@@ -0,0 +1,44 @@
+.pbw-debug-panel {
+  position: absolute;
+  bottom: 6em;
+  left: 2.1em;
+  width: 30%;
+  padding: 0.8em;
+  color: white;
+  font-size: 0.8em;
+  background-color: rgb(0 0 0 / 70%);
```

Comment:
Looks like this is not compatible with Safari, https://caniuse.com/mdn-css_types_color_rgb_mixed_type_parameters

Reply:
@jboix this may be an oversight

Reply:
Most definitely! I though I had changed it.